### PR TITLE
Docs: Update Getting started

### LINKED ISSUE #22431: Docs: Dashboards / Overview page 404
The dashboards overview page (first under features / dashboard) in the menu is not found. Maybe a problem with the menu.yaml file. 

https://grafana.com/docs/grafana/latest/features/dashboard/dashboards/

I remember it working locally or on docs site (not sure) few days ago when I worked on the stat panel docs. I also remember this page was super outdated (in terms of images and text)

### PR DESCRIPTION
**Pull request by oddlittlebird**
**What this PR does / why we need it**: Improves the experience of new users just getting started with Grafana. It provides a path for new users to quickly get results from Grafana as quickly and easily as possible.

**Special notes for your reviewer**: A higher-level Intro to Grafana topic will be added later.

Fixes: #22431

### CODE REVIEW

File: docs/sources/guides/getting_started.md [hunk 1]
```diff
@@ -1,49 +1,86 @@
...
+1. Save your dashboard. Click the **Save dashboard** icon in the top corner of the screen.
+
+ Congratulations, you have gotten started with Grafana! You have a dashboard and are displaying results. Feel free to experiment with what you have built, continue on to add another data source, or explore [Next steps](#next-steps).
 
 ## Add a data source
 
-Before you create your first dashboard, you need to add your data source. Following are the list of instructions to create one.
+{{< docs-imagebox img="/img/docs/v52/sidemenu-datasource.png" max-width="250px" class="docs-image--right docs-image--no-shadow">}}
+
+Before you create your first dashboard, you need to add your data source.
```
Comment (torkelo) [inline docs/sources/guides/getting_started.md]: first real dashboard (as the step before already created a dashboard with test data)

File: docs/sources/guides/getting_started.md [hunk 2]
```diff
@@ -1,49 +1,86 @@
...
+1. On the login page, type `admin` for the username and password.
+1. Change your password. 
+
+> **Note:** We strongly encourage you to follow Grafana best practices and change the default administrator password. Don't forget to record your credentials!
+
+## Add TestData DB data source
+
+1. Click **Add data source**.
+1. Search for or scroll down to TestData DB.
+1. Click **TestData DB** and then click **Save & Test**.
```
Comment (torkelo) [inline docs/sources/guides/getting_started.md]: This is actually not needed, the random walk scenario is already there in a built in data source named "-- Grafana --". But this is still a good thing to have in the guide if someone just wants to play with what visualizations grafana have. But maybe not as a required getting started step.

**Review by torkelo — APPROVED** [review]
Nice, this guide really needed some love!

File: docs/sources/guides/getting_started.md (hunk 1 above)
Comment (oddlittlebird) [reply docs/sources/guides/getting_started.md]: This is theoretically being rewritten by another contributor.

File: docs/sources/guides/getting_started.md (hunk 2 above)
Comment (oddlittlebird) [reply docs/sources/guides/getting_started.md]: So...how do we get visible results without adding the data source?

Comment (oddlittlebird) [reply docs/sources/guides/getting_started.md]: NVM, figured it out.

**Pull request merged**